README.md

```diff
@@ -6,24 +6,19 @@ This is a mobile app project template using [Uno Platform](https://github.com/un
 
 We use `dotnet` project templates to easily create new projects. It simplifies the **project renaming** and supports **conditional inclusions**.
 
-### Installing and uninstalling the template
```

Comment:
I wouldn't change this section.

Reply:
It's confusing more than anything else, it look like steps to be done
1. Install
2. Uninstall